### ISSUE TITLE
Thrust higher than expected at rated wind speed

### ISSUE THREAD

**Issue by verlivkra** [IEAWindSystems/IEA-22-280-RWT#99]
Hi, 

I am working with the IEA 22 MW floating semi in OpenFAST. I ran a case with constant wind speed at 11 m/s. The average rotor thrust is 4.1E3 kN, which is a lot higher than what I can see from the documentation (Figure 20). Similarly, I ran a separate case at 16 m/s and obtained an average thrust of 2.1E3 kN, which is larger than what is shown in the documentation for this wind speed. I am using the libdiscon.dll from ROSCO 2.9.0. Is there other information that could be missing for the controller? I have also attached images showing rotor speed and blade pitch. 

![image](https://github.com/user-attachments/assets/0cd907a8-edfe-4594-b2c1-d2d5d9972057)
![image](https://github.com/user-attachments/assets/df1a67e2-3ce0-4224-aba4-b02f041828d7)

**Comment by fzahle** [comment]
are you using BeamDyn or ElastoDyn, i.e. are you taking into account blade torsion in your simulations? If ElastoDyn, that could explain the higher loads you're observing.

**Comment by verlivkra** [comment]
I am using ElastoDyn. I guess it could influence the loads. However, a colleague ran the monopile version with BeamDyn and ElastoDyn. At 11 m/s (corresponding to 750-1000 s) mean rotor thrust is less than 3.2 MN, while my results indicate 4.1 MN mean thrust. I would expect to see similar results to the monopile. Are the differences between the floater and the bottom-fixed turbine that big? Or could there be some issues with the controller?

![image](https://github.com/user-attachments/assets/76501753-3f9d-4387-9760-04b423c494cc)

**Comment by dzalkind** [comment]
The controller appears to be operating as expected.  Could the difference be that you're looking at the structural rotor thrust (`RotThrust`), which includes the weight of the rotor when the turbine is pitched?  If you look at the aerodynamic rotor thrust `RtFldFxh` in OpenFAST, is there a better match?

**Comment by verlivkra** [comment]
I guess that could be it too. Here is the aerodynamic rotor thrust and the structural rotor thrust for the floating turbine: 
![image](https://github.com/user-attachments/assets/dcb1ca77-4e27-4d8c-8ea2-ca8d0b83e690)

And here are the similar results for the monopile that my colleague ran: 
![image](https://github.com/user-attachments/assets/baf63cd8-b32d-4faa-990a-7f3c242dffbc)

So for the floating turbine, the aerodynamic thrust is about 3.4-3.5 MN, while for the fixed it is about 3-3.05 MN. The platform does have a mean pitch of about 6 deg, but I would maybe expect this to lead to reduced thrust instead of increased.

**Comment by dzalkind** [comment]
Pitching will reduce the aerodynamic thrust, but the structural channel rotates with the turbine and includes the weight of the rotor along that axis.

In BeamDyn, the blades have a torsional DOF, which will effectively pitch the blades and reduce the thrust.

**Comment by verlivkra** [comment]
Considering ElastoDyn blades, and comparing aerodynamic thrust (RtFldFxh) of the bottom-fixed and floating turbine, shouldn't they be quite similar? Or rather the floater having less aerodynamic thrust than the monopile, as opposed to what I see?

**Comment by dzalkind** [comment]
If you use the same model and controller, disable the floating DOFs for one set of simulations, and observe the thrust and pitch of both, are they different?

**Comment by verlivkra** [comment]
I ran four cases, all at 11 m/s wind speed:
1) the monopile
2) the fixed semi (platform dofs disabled), but with all the semi input files
3) the fixed semi with the monopile control input
4) the fixed semi with the monopile control input and the monopile aerodyn-file.
 
It seems like the AeroDyn-settings are causing the differences - when I use monopile AeroDyn, the semi and monopile coincide. Although at a higher BldFxh than I expected:

![image](https://github.com/user-attachments/assets/346a796e-3323-44f9-9fd2-0e24829ffad6)

The two AeroDyn parameters that vary between the fixed and floating turbine are "WakeMod" (11 for the monopile, 12 for the floater) and "UAMod" (4 for the monopile, 3 for the floater). 
I tested with variations of these parameters, and it seems that the variations in "UAMod" are the surce of the differences in thrust: 

![image](https://github.com/user-attachments/assets/cc56e621-032a-4e85-869c-27cea4f64a61)

Is this behavior expected? What is the reason for applying different unsteady aero models for the floating and fixed turbine? And which is more thrustworthy? 

Thanks for helping out!

**Comment by ptrbortolotti** [comment]
hello,
thanks for digging here. You are correct, we ran into this issue already, where for some unclear reason UAMod needs to be set to 4 with BeamDyn and 3 to ElastoDyn, or we run into some instability. We have a dedicated effort together with the University of Massachusetts looking into that, but I still don't know what's going on

**Comment by verlivkra** [comment]
Okay. But I have used ElastoDyn blades for all of these simulations, and they seem stable (based on this one case) for both variations of UAmod. What do you recommend using (if instabilities is not an issue)?

**Comment by ptrbortolotti** [comment]
In this paper https://iopscience.iop.org/article/10.1088/1742-6596/2767/5/052042/meta I've used the fixed bottom model with WakeMod=11, UAMod=4, and BeamDyn. I also ran sims with WakeMod=12, with little change in the results. The loads from OpenFAST matched the other solvers well. Note that switching to UAMod=3 made OpenFAST deviate from the other solvers. I did not run ElastoDyn for the blades.

For the floating model, @dzalkind only modeled the structural dynamics of the blades in ElastoDyn, and had issues sticking to UAMod=4. I believe he had better luck with UAMod=3. However, there was no detailed comparison to other aeroelastic solvers. @dzalkind please confirm that I'm accurate here.

@verlivkra, is this matching what you observe?

Thank you for conducting these extra investigations

**Comment by verlivkra** [comment]
Thanks for this reference. For the bottom-fixed turbine, I obtain a mean aerodynamic thrust of about 3.25 MN at 11 m/s constant wind when using ElastoDyn. With BeamDyn leading to a reduction in thrust, I guess that it could be matching the results you provide in the paper (my colleague's results in thrust with BeamDyn seem to match the ones you report). 

I am trying to run a turbulent wind case with mean wind speed 16 m/s for the floating turbine with ED blades now, and it seems to be going ok, although some warnings appear. 

![image](https://github.com/user-attachments/assets/22a1858d-5b99-4e96-a4e8-a43e16679752)

As you have verified UAMod = 4 against other codes, it is tempting to continue trying with this setting for the floating turbine. however the time series look a lot more messy for UAMod = 4. Perhaps this is what you refer to as instability? 

![image](https://github.com/user-attachments/assets/cb1b22c6-b2fe-4e54-8f6c-8d656a77f8da)

Perhaps my knowledge about unsteady aerodynamics has some shortcomings, but I am somewhat surprised that there is such a large difference in the mean thrust with switching between unsteady aerodynamics models.

**Comment by ptrbortolotti** [comment]
you are correct that UAMod 3 and 4 should not lead to this large variation in load, and we are investigating what's going on. I don't have an answer for you yet unfortunately. I will say that the validation described here https://doi.org/10.5194/wes-9-1791-2024 also used BeamDyn and UAMod = 4 (and WakeMod=2).

If I were to pick a combination, I'd go ahead with BeamDyn + UAMod=4. I understand that BeamDyn is still very slow, but the release of OpenFAST 5.0 is near, and it will bring a much desired speed up effect

**Comment by verlivkra** [comment]
Thanks a lot for clarifying and providing informative feedback. How does the timeline for the release of the tight coupling look like? 

FYI. I ran a simulation of the floating turbine (platform dofs enabled) with ElastoDyn and UAMod = 4 (WakeMod 12) with 16 m/s turbulent wind and waves, and the simulations went through. 

The mean thrust is slightly different among the models (4%), but not as significant as seen for the fixed platform with constant wind at 11 m/s (7%). 

![image](https://github.com/user-attachments/assets/4c7b0697-7fd9-4737-b34e-eb7e62259e76)

The rotor thrust with UAMod = 4 looks less smooth, though:

![image](https://github.com/user-attachments/assets/daa731e1-a36f-44ca-aff9-98cba17f8529)

**Comment by ptrbortolotti** [comment]
Let's start from the end: the warning is about node 3, very close to blade root. I wouldn't worry too much about it. You can adjust `UAStartRad` if you want to suppress the warning.

The pull request for tight coupling is sitting in the queue https://github.com/OpenFAST/openfast/pull/2439

As far as our tests show, it is ready to be merged and you are free to be an early tester. Note that the input files are quite different from OpenFAST v3.5.3, so you'll need to update your model. 

I attach some files for the fixed-bottom version of the model that I generated while testing the new OpenFAST

[turb_iea22_0deg_TC.zip](https://github.com/user-attachments/files/17651511/turb_iea22_0deg_TC.zip)

**Comment by ptrbortolotti** [comment]
Closing this issue for now, please reopen if problems persist
